charts/metrics/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: metrics
 description: Observe metrics collection
 type: application
-version: 0.3.5
+version: 0.3.6
 dependencies:
   - name: grafana-agent
-    version: 0.10.0
+    version: operator-0.3.10
```

Comment:
looks questionable :eyes:

Reply:
Need to fix the [regex](https://github.com/observeinc/helm-charts/blob/main/updatecli/external/grafana.yml#L10C23-L10C23) so it will match `grafana-agent-x.y.z` and not include `grafana-agent-operator-x.y.z`